File: front/service/src/main/java/com/coinhunt/front/data/CompletedGameEntry.java
package com.coinhunt.front.data;

import lombok.AllArgsConstructor;
import lombok.Getter;

@AllArgsConstructor
@Getter
public class CompletedGameEntry {

	private final String userId;

	private final long totalTimeInMilliseconds;

	private final long startTimeEpochMilli;
}

File: games/src/main/kotlin/com/coinhunt/games/persistence/domain/documents/CompletedGameEntry.kt
package com.coinhunt.games.persistence.domain.documents

import org.bson.types.ObjectId
import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document
import java.time.Instant

@Document
data class CompletedGameEntry(
    val userId: Long,
    val totalTimeInMilliseconds: Long,
    val startTime: Instant,
    @Id val id: ObjectId? = ObjectId.get()
)
